config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
+development:
+  <<: *default
+  database: db/development.sqlite3
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: db/test.sqlite3
+
+production:
+  <<: *default
+  database: db/production.sqlite3
```

Comment:
- [ ] According to the requirement of this project, it was stated that the Database adapter should use `PostgreSQL` but currently in your project, we have the SQLite3 as the adapter storing your database. I would kindly suggest you store your database using PostgreSQL in order to stick to the project requirement.

![data](https://github.com/najibullahjafari/ruby_social_media/assets/78634841/0db100be-f08f-4999-bf20-ec529f21c0c7)